Examples/WebViewExample/Podfile

```diff
@@ -0,0 +1,8 @@
+use_frameworks!
+
+pod "Branch", path: "../.."
```

Comment:
I can add a comment explaining what's going on. I think using paths or file URLs is a common idiom with contemporary package managers, e.g.:

```Ruby
gem "fastlane", path: "~/github/fastlane/fastlane"
```

From the package.json of each example in the react-native-branch repo (except those that use v1.1).
```json
"react-native-branch": "file:../.."
```